### PR TITLE
App updates

### PR DESCRIPTION
**Pull request by freiburgermsu**
1. Updated docs, including the removal of all LaTeX and the replacement of equations with images
2. Updated name for the members argument of the App

I will begin curating the scores and investigating additional App features.

How is the App updated from these edits in GitHub? The App maintainer on another of my projects has abandoned updating the App so I must update it.

### CODE REVIEW

File: .idea/vcs.xml
Comment (pranjan77) [inline .idea/vcs.xml:1]: can you put files like these in your .gitignore?

File: ui/narrative/methods/run_microbial_interactions/spec.json [hunk 1]
```diff
@@ -47,8 +46,8 @@
                     "narrative_system_variable": "workspace_id",
                     "target_property": "workspace_id"
                 },{
-                    "input_parameter": "input_models",
-                    "target_property": "input_models"
+                    "input_parameter": "member_models",
+                    "target_property": "member_models"
```
Comment (pranjan77) [inline ui/narrative/methods/run_microbial_interactions/spec.json:50]: All other instances of input_models like the following needs to change https://github.com/pranjan77/microbial_interactions/blob/main/lib/microbial_interactions/microbial_interactionsImpl.py#L76

File: ui/narrative/methods/run_microbial_interactions/display.yaml [hunk 2]
```diff
@@ -42,5 +42,117 @@ parameters :
...
+    
+    <div class="publication">
+        <h3>Related Publications</h3>
+        <ol>
+            <li id="smetana">
+                Zelezniak, A., Andrejev, S., Ponomarova, O., Mende, D. R., Bork, P., &#38; Patil, K. R. (2015). 
+                Metabolic dependencies drive species co-occurrence in diverse microbial communities. <i>Proceedings of the National Academy of Sciences of the United States of America</i>, 
+                <i>112</i>(20), 6449-6454. <a href="https://www.pnas.org/doi/abs/10.1073/pnas.1421834112">https://www.pnas.org/doi/abs/10.1073/pnas.1421834112</a>
+            </li>
+            <li id="kbase">
```
Comment (pranjan77) [inline ui/narrative/methods/run_microbial_interactions/display.yaml]: Publications need to be listed like here https://github.com/kbaseapps/kb_Msuite/blob/master/ui/narrative/methods/run_checkM_lineage_wf/display.yaml#L75

**Review by pranjan77 — CHANGES_REQUESTED** [review]
Mostly looks good but needs few changes.

**Comment by freiburgermsu** [comment]
[This HTML report object](https://github.com/pranjan77/microbial_interactions/blob/main/lib/microbial_interactions/microbial_interactionsImpl.py#L88) is never used in your code; how is the HTML report generated in the App?

**Comment by pranjan77** [comment]
> [This HTML report object](https://github.com/pranjan77/microbial_interactions/blob/main/lib/microbial_interactions/microbial_interactionsImpl.py#L88) is never used in your code; how is the HTML report generated in the App?


result_dir has the index.html. Entire result_dir is sent to create_html_report. https://github.com/pranjan77/microbial_interactions/blob/main/lib/microbial_interactions/microbial_interactionsImpl.py#L83C9-L84C1   The advantage of doing it this way is your result_dir can have the entire directory structure you need to support the html page. You can have js, css, images everything in result_dir and it will work.

**Comment by freiburgermsu** [comment]
I don't understand the HTML workflow; I'll ask for clarification tomorrow.

I believe all of the critiques are corrected in the most recent code.

**Review by pranjan77 — COMMENTED** [review]
Looks good.

**Pull request merged**